building_energy_standards_data/database_files/hvac_minimum_requirement_furnaces_90_1_prm.json

```diff
@@ -0,0 +1,705 @@
+        "minimum_thermal_efficiency": null,
+        "minimum_combustion_efficiency": 0.8,
+        "standby_mode_power": null,
+        "off_mode_power": null,
+        "annotation": "From 90.1-2016 Table G3.5.5"
+    },
+    {
+        "template": "90.1-PRM-2019",
+        "equipment_type": "Warm Air Furnace",
+        "fuel_type": "NaturalGas",
+        "electric_power_phase": null,
+        "minimum_capacity": 0,
+        "maximum_capacity": 224999.99,
+        "minimum_combo_unit_cooling_capacity": null,
+        "maximum_combo_unit_cooling_capacity": null,
+        "subtype": null,
+        "start_date": "9/9/1919",
+        "end_date": "9/9/2999",
+        "minimum_annual_fuel_utilization_efficiency": 0.78,
+        "minimum_thermal_efficiency": 0.8,
+        "minimum_combustion_efficiency": null,
+        "standby_mode_power": null,
+        "off_mode_power": null,
+        "annotation": "From 90.1-2019 Table G3.5.5"
+    },
+    {
+        "template": "90.1-PRM-2019",
+        "equipment_type": "Warm Air Furnace",
+        "fuel_type": "NaturalGas",
+        "electric_power_phase": null,
+        "minimum_capacity": 225000,
+        "maximum_capacity": 999999,
+        "minimum_combo_unit_cooling_capacity": null,
+        "maximum_combo_unit_cooling_capacity": null,
+        "subtype": null,
+        "start_date": "9/9/1919",
+        "end_date": "9/9/2999",
+        "minimum_annual_fuel_utilization_efficiency": null,
+        "minimum_thermal_efficiency": null,
+        "minimum_combustion_efficiency": 0.8,
+        "standby_mode_power": null,
+        "off_mode_power": null,
+        "annotation": "From 90.1-2019 Table G3.5.5"
+    },
+    {
+        "template": "90.1-PRM-2019",
+        "equipment_type": "Warm Air Unit Heaters",
+        "fuel_type": "NaturalGas",
+        "electric_power_phase": null,
+        "minimum_capacity": 0,
+        "maximum_capacity": 999999,
+        "minimum_combo_unit_cooling_capacity": null,
+        "maximum_combo_unit_cooling_capacity": null,
+        "subtype": null,
+        "start_date": "9/9/1919",
+        "end_date": "9/9/2999",
+        "minimum_annual_fuel_utilization_efficiency": null,
+        "minimum_thermal_efficiency": null,
+        "minimum_combustion_efficiency": 0.8,
+        "standby_mode_power": null,
+        "off_mode_power": null,
+        "annotation": "From 90.1-2019 Table G3.5.5"
+    },
+    {
+        "template": "90.1-PRM-2022",
+        "equipment_type": "Warm Air Furnace",
+        "fuel_type": "NaturalGas",
+        "electric_power_phase": null,
+        "minimum_capacity": 0,
+        "maximum_capacity": 224999.99,
+        "minimum_combo_unit_cooling_capacity": null,
+        "maximum_combo_unit_cooling_capacity": null,
+        "subtype": null,
+        "start_date": "9/9/1919",
+        "end_date": "9/9/2999",
+        "minimum_annual_fuel_utilization_efficiency": 0.78,
+        "minimum_thermal_efficiency": 0.8,
```

Comment:
This is indeed correct, but I wonder why he leave the choice to the user. I'm checking to see if they are equivalent.

Reply:
This is one of the footnotes related to this from 90.1-2016:
* Combination units not covered by the U.S. Department of Energy Code of Federal Regulations 10 CFR 430 (three-phase power or cooling capacity greater than or equal to 65,000 Btu/h) may comply with either rating.